### PR TITLE
doc: reword announcement about upcoming 1.0.0

### LINKED ISSUE #1795: doc: remove/reword announcement about upcoming 1.0.0 on release
**Description of issue or feature request**:
Once we have released 1.0.0 it no longer makes sense to say that we are about to release 1.0.0. We should remove/reword the corresponding docs parts shortly before we cut the 1.0.0 release.

**Current behavior**:
- Main [README has an "important notice"](https://github.com/theupdateframework/python-tuf/blame/e0736d6d75e344b10ac128e5798cde3136ccc9cf/README.md#L10-L14) about upcoming 1.0.0 release
- [docs/1.0.0-ANNOUNCEMENT.md](https://github.com/theupdateframework/python-tuf/blob/e0736d6d75e344b10ac128e5798cde3136ccc9cf/docs/1.0.0-ANNOUNCEMENT.md) is written in a way that suggest that the 1.0.0 release is in the future

**Expected behavior**:
- Remove important notice about upcoming 1.0.0 release from main readme
- Reword docs/1.0.0-ANNOUNCEMENT.md to not sound outdated after release (or just remove)

### PR DESCRIPTION
**Pull request by lukpueh**
NOTE: Should be merged directly before release (coordinate with #1851)

Fixes #1795 

**Description of the changes being introduced by the pull request**:
Change docs in preparation of close v1.0.0 release.

- Remove important notice about upcoming 1.0.0 release from README
- Reword 1.0.0-ANNOUNCEMENT.md to not sound outdated after release

**Please verify and check that the pull request fulfills the following
requirements**:

- [ ] The code follows the [Code Style Guidelines](https://github.com/secure-systems-lab/code-style-guidelines#code-style-guidelines)
- [ ] Tests have been added for the bug fix or new feature
- [ ] Docs have been added for the bug fix or new feature

### CODE REVIEW

File: docs/1.0.0-ANNOUNCEMENT.md [hunk 1]
```diff
@@ -1,8 +1,7 @@
 # Announcing TUF 1.0.0
 
-In the past year we have made an effort to revise, redesign and rewrite this
-python-tuf reference implementation, and we are very excited to announce a
-stable 1.0.0 release scheduled for January 2022. The release *will* include:
+Python-TUF v1.0.0 is a rewritten stable reference implementation of the TUF
+specification, which *includes*:
```
Comment (joshuagl) [inline docs/1.0.0-ANNOUNCEMENT.md]:
```suggestion
specification, which *currently* includes:
```
opposite to the "*does not yet* include a new *repository tool*" statement

File: docs/1.0.0-ANNOUNCEMENT.md [hunk 2]
```diff
@@ -11,7 +10,7 @@ stable 1.0.0 release scheduled for January 2022. The release *will* include:
   for the legacy client updater
 
 As discussed in [ADR 2](https://github.com/theupdateframework/python-tuf/blob/develop/docs/adr/0002-pre-1-0-deprecation-strategy.md), this
-release *will not* include any legacy code, as its maintenance has become
+release *does not* include any legacy code, as its maintenance has become
 infeasible for the python-tuf team. The pre-1.0.0 deprecation strategy from ADR
 2 applies as follows:
```
Comment (joshuagl) [inline docs/1.0.0-ANNOUNCEMENT.md:21]: Now that this code has been removed, should this statement be moved down? The next paragraph seems more important.

File: docs/1.0.0-ANNOUNCEMENT.md [hunk 3]
```diff
@@ -22,13 +21,11 @@ processes). Note that there may be delays due to the lack of developer resources
 for reviewing such pull requests.*
 
 For the reasons outlined in [ADR 10](https://github.com/theupdateframework/python-tuf/blob/develop/docs/adr/0010-repository-library-design.md
-), this release *will not yet* include a new *repository tool*. However, the new
+), this release *does not yet* include a new *repository tool*. However, the new
 *metadata API* makes it easy to replicate the desired functionality tailored to
 the specific needs of any given repository (see *Migration* for details).
```
Comment (joshuagl) [inline docs/1.0.0-ANNOUNCEMENT.md]:
```suggestion
), this release *does not yet* include *repository tool*-like functionality. However, the new
*metadata API* makes it easy to replicate the desired functionality tailored to
the specific needs of any given repository (see *Migration* for details).
```
Nit: we don't want to create a new repository-tool, we want to create something better. This edit tries to emphasise that we want to provide repository-tool-like functionality

**Review by joshuagl — COMMENTED** [review]
Thanks. This looks good. I made a few minor suggestions, WDYT?

**Comment by lukpueh** [comment]
Thanks for the review, @joshuagl! All your suggestions make sense and are now in https://github.com/theupdateframework/python-tuf/pull/1869/commits/399a4823b49b2ec7232c75bc1eb225d2d831b6a8

**Review by joshuagl — APPROVED** [review]
Thanks for the quick edits, LGTM

**Comment by lukpueh** [comment]
Superseded by #1873

**Pull request closed without merging**